README.md

```diff
@@ -1,9 +1,9 @@
 # Operators for Qiskit
```

Comment:
I'd like to see this have more detail in it - for example...how do you build the operator?, etc....more or less info about how to use and contribute to this operator or test it, etc... your link to the operators-examples/qiskit-dev-operator has some good info in it on how to run and test so that is good